src/ui/components/Auth/Login.jsx

```diff
@@ -0,0 +1,55 @@
+import './auth.css';
+import React, { memo, useCallback, useRef } from 'react';
+import { TextField, Button } from '@material-ui/core';
+import axios from 'axios';
+import { useQueryCache } from 'react-query';
+
+const Login = () => {
+  const cache = useQueryCache();
```

Comment:
I tried looking up what this hook does and the docs for it seem to say it's getting deprecated as well, though it looks like it's only really changing names so it shouldn't be a problem to change.

Reply:
Updated to use QueryClient